audit_exceptions/github_prerelease_allowlist.json

```diff
@@ -6,6 +6,7 @@
   "grt": "0.2.4",
   "joshuto": "all",
   "lha": "all",
+  "libesedb": "all",
```

Comment:
Why would we want to make an exception for this software?

Reply:
so far all the releases are all marked as pre-release

<img width="970" alt="image" src="https://github.com/user-attachments/assets/a6befa8d-34ca-4f83-b90f-8816fa454eca" />

Reply:
I could just change to allow the current ones though, also gonna raise an ticket about it.